core-it-suite/src/test/resources/mng-5581-lifecycle-mapping-delegate/extension/src/main/java/org/apache/maven/its/mng5581/lifecyclemappingdelegate/TestLifecycleMappingDelegate.java

```diff
@@ -57,7 +57,7 @@ public Map<String, List<MojoExecution>> calculateLifecycleMappings(
             throws PluginNotFoundException, PluginResolutionException, PluginDescriptorParsingException,
                     MojoNotFoundException, InvalidPluginDescriptorException {
 
-        Map<String, List<MojoExecution>> pluginExecutions = new LinkedHashMap<String, List<MojoExecution>>();
+        Map<String, List<MojoExecution>> pluginExecutions = new LinkedHashMap<>();
```

Comment:
This failed with
```
[INFO] Compiling 1 source file to /home/runner/work/maven-integration-testing/maven-integration-testing/core-it-suite/target/test-classes/mng-5581-lifecycle-mapping-delegate/extension/target/classes
[INFO] -------------------------------------------------------------
Error:  COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
Error:  /home/runner/work/maven-integration-testing/maven-integration-testing/core-it-suite/target/test-classes/mng-5581-lifecycle-mapping-delegate/extension/src/main/java/org/apache/maven/its/mng5581/lifecyclemappingdelegate/TestLifecycleMappingDelegate.java:[60,79] diamond operator is not supported in -source 1.5
  (use -source 7 or higher to enable diamond operator)
```
Easiest thing to do then is to restore the types.
```suggestion
        Map<String, List<MojoExecution>> pluginExecutions = new LinkedHashMap<String, List<MojoExecution>>();
```